README.md

```diff
@@ -12,12 +12,20 @@ The project is currently under development and all documentation is aimed at pro
 
 ### Instructions
  1. Clone this repository to your machine
- 2. `cd` into the repo directory and set up a virtual environment per the instructions [here](https://docs.getdbt.com/docs/core/pip-install#using-virtual-environments).  Note that if you're using VS Code, the vscode config in the repo will automatically source the virtualenv each time you open a new terminal in the project
- 3. Create a .env file in your repo and add a PYTHONPATH entry for your virtual env (e.g. `PYTHONPATH="~/dbt-synthea/dbt-env/bin/python"`)
+ 2. `cd` into the repo directory and set up a virtual environment:
+ ```bash
+ $ python3 -m venv dbt-env
+ ```
+ - Note that if you're using VS Code, once you set this env as your preferred interpreter for the project, the vscode config in the repo will automatically source the virtualenv each time you open a new terminal in the project.  Otherwise, each time you open a new terminal to use dbt for this project, run:
+```bash
+$ source dbt-env/bin/activate         # activate the environment for Mac and Linux OR
+$ dbt-env\Scripts\activate            # activate the environment for Windows
+```
+ 3. Create a .env file in your repo (`touch .env`) and add a PYTHONPATH entry for your virtual env (for example, if you created your repo in your computer's home directory, the entry will read as: `PYTHONPATH="~/dbt-synthea/dbt-env/bin/python"`)
```

Comment:
I think, we can include `source .env` to execute `.env` file in the current shell environment.

Reply:
that's not needed, this is just for VS Code to locate the virtual env. i clarified this in the instructions!